holidays/countries/bhutan.py

```diff
@@ -0,0 +1,91 @@
+#  holidays
+#  --------
+#  A fast, efficient Python library for generating country, province and state
+#  specific sets of holidays on the fly. It aims to make determining whether a
+#  specific date is a holiday as fast and flexible as possible.
+#
+#  Authors: Vacanza Team and individual contributors (see AUTHORS.md file)
+#           dr-prodigy <dr.prodigy.github@gmail.com> (c) 2017-2023
+#           ryanss <ryanssdev@icloud.com> (c) 2014-2017
+#  Website: https://github.com/vacanza/holidays
+#  License: MIT (see LICENSE file)
+
+from gettext import gettext as tr
+
+from holidays.groups import TibetanCalendarHolidays
+from holidays.holiday_base import HolidayBase
+
+
+class Bhutan(HolidayBase, TibetanCalendarHolidays):
+    """Bhutan holidays.
+
+    References:
+        * <https://www.wipo.int/wipolex/en/legislation/details/16762>
+        * <https://www.bhutan.travel/travel-guide/bhutanese-holidays-and-festivals>
+    """
+
+    country = "BT"
+    default_language = "dz_BT"
+    supported_languages = ("dz_BT", "en_US")
+    start_year = 1970
+
+    def __init__(self, *args, **kwargs):
+        TibetanCalendarHolidays.__init__(self)
+        super().__init__(*args, **kwargs)
+
+    def _populate_public_holidays(self):
+        # Nyilo.
+        self._add_holiday_jan_2(tr("ཉི་ལོག་"))
```

Comment:
This holiday is obviously celebrated according to some traditional solar calendar, so it does not always fall on January 2 in the Gregorian calendar.